### PR TITLE
Unit control page

### PR DESCRIPTION
**Pull request by vitaly-furman**

### CODE REVIEW

**Comment by lustigy96** [comment]
before merging, check client across real backend locally

File: src/pages/MalshabSchedule/components/BasicTable/BasicTable.styles.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  root: {
+    height: '77.5%',
+    direction: 'rtl',
+  },
+  message: {
+    fontWeight: '600',
+    fontSize: '1.5rem',
+    textAlign: 'center',
+    color: 'lightgrey',
+  },
```
Comment (carmeltsadok) [inline src/pages/MalshabSchedule/components/BasicTable/BasicTable.styles.js:13]: use `NoObjectsToShow` component

File: src/pages/MalshabSchedule/components/NodeGroups/NodeGroups.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          {unitNodesGroups.map((nodeGroup) => (
+            <NodeGroupCard
+              nodeGroup={nodeGroup}
+              selectedDate={selectedDate}
+              key={nodeGroup.id}
+              setChoosenNodeGroup={setChoosenNodeGroup}
+            />
+          ))}
+        </div>
+      ) : <div className={classes.noNodesGroups}>{t('text.noNodesGroups')}</div>}
```
Comment (carmeltsadok) [inline src/pages/MalshabSchedule/components/NodeGroups/NodeGroups.jsx:23]: use `NoObjectsToShow` component

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: src/pages/MalshabSchedule/components/SelectScheduling/SelectScheduling.styles.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    fill: 'white',
+    backgroundColor: 'white',
+    width: '8rem',
+    height: '2.5rem',
+    borderRadius: '8px',
+    padding: '0 0.6em',
```
Comment (lustigy96) [inline src/pages/MalshabSchedule/components/SelectScheduling/SelectScheduling.styles.js:10]: add border radius

File: src/pages/MalshabSchedule/components/SelectBox/SelectBox.styles.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& label.Mui-focused': {
+      color: theme.palette.textField.focused,
+    },
```
Comment (lustigy96) [inline src/pages/MalshabSchedule/components/SelectBox/SelectBox.styles.js:7]: add border radius

**Review by lustigy96 — CHANGES_REQUESTED** [review]

**Pull request merged**